src/ParameterParser.h

```diff
@@ -403,6 +403,32 @@ namespace snowcrash {
             }
         }
 
+        static size_t matchBracket(const mdp::ByteBuffer& text) {
```

Comment:
I think this function? Not sure if it does the same thing though.

 https://github.com/apiaryio/snowcrash/blob/master/src/SignatureSectionProcessor.h#L363